release/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+# This is intended to be run from the  root directory. `release/build.sh`
+set -e
+
+GIT_ROOT=`git rev-parse --show-toplevel`
+
+cd $GIT_ROOT # We need to start from repository root
+
+#####################################################
+#      Generating Artifacts opensearch-cli                      #
+#                                                   #
+#####################################################
+
+
+echo 'Generating artifacts'
+goreleaser --snapshot --skip-publish --rm-dist
+
+rm -rf dist/opensearch-cli_*
```

Comment:
Why are we deleting these here?

Reply:
This tool generated more than what we wanted I couldn't figure out a way to disable generation of those unwanted files. Hence, deleting the unwanted files. I will add this as comment too.